### ISSUE TITLE
name 'sum_loss_op' is not defined

### ISSUE THREAD

**Issue by tianyaaz** [csmhwu/tensorflowbook#9]
5.6章节 里5.6.5迭代训练中
_, summary_str, loss = sess.run([optimizer,sum_loss_op,loss_function], feed_dict={x: xs, y: ys})，
其中sum_loss_op前面并没有定义，我照着敲的时候执行就报错了
